### PR TITLE
added minimumLineHeight and applied ceil on height

### PR DESCRIPTION
**Pull request by demonar**
added minimumLineHeight to be able to handle emojis properly and applied ceil on intrinsicContentSize() to avoid glitches created by decimal pixels

### CODE REVIEW

File: ActiveLabel/ActiveLabel.swift [hunk 1]
```diff
@@ -39,7 +39,9 @@ public protocol ActiveLabelDelegate: class {
     @IBInspectable public var lineSpacing: Float = 0 {
         didSet { updateTextStorage(parseText: false) }
     }
-
+    @IBInspectable public var minimumLineHeight: Float = 0 {
```
Comment (poolqf) [inline ActiveLabel/ActiveLabel.swift]: what about CGFloat ?

File: ActiveLabel/ActiveLabel.swift [hunk 2]
```diff
@@ -306,7 +308,7 @@ public protocol ActiveLabelDelegate: class {
         paragraphStyle.lineBreakMode = NSLineBreakMode.ByWordWrapping
         paragraphStyle.alignment = textAlignment
         paragraphStyle.lineSpacing = CGFloat(lineSpacing)
-        
+        paragraphStyle.minimumLineHeight = CGFloat(minimumLineHeight > 0 ? minimumLineHeight: self.font.pointSize * 1.14)
```
Comment (poolqf) [inline ActiveLabel/ActiveLabel.swift]: Remove the CGFloat around all the expression after the `=`

**Comment by poolqf** [comment]
Hi @demonar, any update?

**Comment by demonar** [comment]
Updated 😄

**Pull request merged**